src/TodoListItem.jsx

```diff
@@ -1,6 +1,11 @@
 // eslint-disable-next-line react/prop-types
-export default function TodoListItem({ title }) {
+export default function TodoListItem({ title, id, onRemoveTodo }) {
```

Comment:
This function could accept the listItem and use its title and id, rather than accepting the title and id directly. The way it is here isn't wrong, but I thought you should know about this as an alternative option.